rust-runtime/aws-smithy-runtime-api/src/client/runtime_components.rs

```diff
@@ -380,12 +380,40 @@ declare_runtime_components! {
     }
 }
 
+impl From<RuntimeComponents> for RuntimeComponentsBuilder {
+    fn from(rc: RuntimeComponents) -> Self {
+        RuntimeComponentsBuilder {
+            builder_name: "converted from RuntimeComponents",
```

Comment:
This one should be removed in favor of `from_runtime_components`

Reply:
Removed in 027c2cd.